repo/packages/K/kafka/500/resource.json

```diff
@@ -0,0 +1,62 @@
+{
+  "assets": {
+    "uris": {
+      "jre-tar-gz": "https://downloads.mesosphere.com/java/server-jre-8u192-linux-x64.tar.gz",
+      "libmesos-bundle-tar-gz": "https://downloads.mesosphere.com/libmesos-bundle/libmesos-bundle-1.12.0.tar.gz",
+      "bootstrap-zip": "https://downloads.mesosphere.com/dcos-commons/artifacts/0.55.2/bootstrap.zip",
+      "kafka-tgz": "https://downloads.mesosphere.com/kafka/assets/kafka_2.12-1.1.1.tgz",
+      "kafka-jre-tar-gz": "https://downloads.mesosphere.com/java/server-jre-8u192-linux-x64.tar.gz",
+      "kafka-scheduler-zip": "https://downloads.mesosphere.com/kafka/assets/2.4.0-1.1.1/kafka-scheduler.zip",
+      "kafka-statsd-jar": "http://downloads.mesosphere.com/kafka/assets/kafka-statsd-metrics2-0.5.3.jar",
+      "statsd-client-jar": "http://downloads.mesosphere.com/kafka/assets/java-dogstatsd-client-2.3.jar",
+      "setup-helper-zip": "https://downloads.mesosphere.com/kafka/assets/2.4.0-1.1.1/setup-helper.zip",
+      "zookeeper-client-jar": "http://downloads.mesosphere.com/kafka/assets/zookeeper-3.4.10.1.jar",
+      "custom-kafka-principal-jar": "http://downloads.mesosphere.com/kafka/assets/kafka-custom-principal-builder-1.0.0.jar"
+    }
+  },
+  "images": {
+    "icon-small": "https://downloads.mesosphere.com/assets/universe/000/kafka-icon-small.png",
+    "icon-medium": "https://downloads.mesosphere.com/assets/universe/000/kafka-icon-medium.png",
+    "icon-large": "https://downloads.mesosphere.com/assets/universe/000/kafka-icon-large.png"
+  },
+  "cli": {
+    "binaries": {
+      "darwin": {
+        "x86-64": {
+          "contentHash": [
+            {
+              "algo": "sha256",
+              "value": "37ebe940d0f09afeb5b9e858291a94f2c8767bfa4da401604b89b644b595fe19"
+            }
+          ],
+          "kind": "executable",
+          "url": "https://downloads.mesosphere.com/kafka/assets/2.4.0-1.1.1/dcos-service-cli-darwin"
+        }
+      },
+      "linux": {
+        "x86-64": {
+          "contentHash": [
+            {
+              "algo": "sha256",
+              "value": "fe70ce0cf7e0ea960f3cb162de6bbb8d2d07a9deb4ab89f41e36b82f6c60c7cc"
+            }
+          ],
+          "kind": "executable",
+          "url": "https://downloads.mesosphere.com/kafka/assets/2.4.0-1.1.1/dcos-service-cli-linux"
```

Comment:
Have we verified that the CLIs work?

Reply:
Manually verified that the Linux CLI (`dcos-service-cli-linux`) works:

```
$ curl -fsSL -O https://downloads.mesosphere.com/kafka/assets/2.4.0-1.1.1/dcos-service-cli-linux
$ chmod +x dcos-service-cli-linux 
$ ./dcos-service-cli-linux 
Must have at least one argument for the CLI module name: ./dcos-service-cli-linux <modname>
$ ./dcos-service-cli-linux kafka

usage: dcos kafka [<flags>] <command>


Flags:
  -h, --help          Show context-sensitive help.
  -v, --verbose       Enable extra logging of requests/responses
      --name="kafka"  Name of the service instance to query

Commands:
  help [<command> ...]
    Show help.


  config list
    [Deprecated (TBR 1.11, -> debug config list)]List IDs of all available configurations


  config show <config_id>
    [Deprecated (TBR 1.11, -> debug config show)] Display a specified configuration


  config target
    [Deprecated (TBR 1.11, -> debug config target)] Display the target configuration


  config target_id
    [Deprectaed (TBR 1.11, -> debug config target_id)] List ID of the target configuration


  debug config list
    List IDs of all available configurations


  debug config show <config_id>
    Display a specified configuration


  debug config target
    Display the target configuration


  debug config target_id
    List ID of the target configuration


  debug state framework_id
    Display the Mesos framework ID


  debug state properties
    List names of all custom properties


  debug state property <name>
    Display the content of a specified property


  debug state refresh_cache
    Refresh the state cache, used for debugging


  debug pod pause [<flags>] <pod>
    Pauses a pod's tasks for debugging

    -t, --tasks=TASKS ...  List of specific tasks to be paused, otherwise the entire pod


  debug pod resume [<flags>] <pod>
    Resumes a pod's normal execution following a pause command

    -t, --tasks=TASKS ...  List of specific tasks to be resumed, otherwise the entire pod


  describe
    View the configuration for this service


  endpoints [<name>]
    View client endpoints


  plan list
    Show all plans for this service


  plan status [<flags>] <plan>
    Display the status of the plan with the provided plan name

    --json  Show raw JSON response instead of user-friendly tree


  plan start [<flags>] <plan>
    Start the plan with the provided name and any optional plan arguments

    -p, --params=PARAMS ...  Envvar definition in VAR=value form; can be repeated for multiple variables


  plan stop <plan>
    Stop the running plan with the provided name


  plan pause <plan> [<phase>]
    Pause the plan, or a specific phase in that plan with the provided phase name (or UUID)


  plan resume <plan> [<phase>]
    Resume the plan, or a specific phase in that plan with the provided phase name (or UUID)


  plan force-restart <plan> [<phase> [<step>]]
    Restart the plan with the provided name, or a specific phase in the plan with the provided name, or a specific step in a phase of the plan with
    the provided step name.


  plan force-complete <plan> <phase> <step>
    Force complete a specific step in the provided phase. Example uses include the following: Abort a sidecar operation due to observed failure or
    known required manual preparation that was not performed


  pod list
    Display the list of known pod instances


  pod status [<flags>] [<pod>]
    Display the status for tasks in one pod or all pods

    --json  Show raw JSON response instead of user-friendly tree


  pod info <pod>
    Display the full state information for tasks in a pod


  pod restart <pod>
    Restarts a given pod without moving it to a new agent


  pod replace <pod>
    Destroys a given pod and moves it to a new agent


  state framework_id
    [Deprecated (TBR 1.11, -> debug state framework_id)] Display the Mesos framework ID


  state properties
    [Deprecated (TBR 1.11, -> debug state properties)] List names of all custom properties


  state property <name>
    [Deprecated (TBR 1.11, -> debug state property)] Display the content of a specified property


  state refresh_cache
    [Deprecated (TBR 1.11, -> debug state refresh_cache)] Refresh the state cache, used for debugging


  update start [<flags>]
    Launches an update operation

    --options=OPTIONS  Path to a JSON file that contains customized package installation options
    --package-version=PACKAGE-VERSION  
                       The desired package version
    --replace          Replace the existing configuration in whole. Otherwise, the existing configuration and options are merged.


  update force-complete <phase> <step>
    Force complete a specific step in the provided phase


  update force-restart [<phase> [<step>]]
    Restart update plan, or specific step in the provided phase


  update package-versions
    View a list of available package versions to downgrade or upgrade to


  update pause
    Pause update plan


  update resume
    Resume update plan


  update status [<flags>]
    View status of a running update

    --json  Show raw JSON response instead of user-friendly tree


  broker list
    Lists all broker ids in the service


  broker get [<id>]
    View a single broker


  topic create [<flags>] [<topic>]
    Creates a new topic

    -p, --partitions=1   Number of partitions
    -r, --replication=3  Replication factor


  topic delete [<topic>]
    Deletes an existing topic


  topic describe [<topic>]
    Describes a single existing topic


  topic list
    Lists all available topics


  topic offsets [<flags>] [<topic>]
    Returns the current offset counts for a topic

    --time="last"  Offset for the topic: 'first'/'last'/timestamp_millis


  topic partitions [<topic> [<count>]]
    Alters partition count for an existing topic


  topic producer_test [<topic> [<messages>]]
    Produces some test messages against a topic


  topic producer_test_tls [<topic> [<messages>]]
    Produces some test messages against a topic over a TLS connection


  topic unavailable_partitions
    Gets info for any unavailable partitions


  topic under_replicated_partitions
    Gets info for any under-replicated partitions
```